hawkbit-security-core/src/test/java/org/eclipse/hawkbit/util/IpUtilTest.java

```diff
@@ -11,9 +11,7 @@
 
 import static com.google.common.net.HttpHeaders.X_FORWARDED_FOR;
 import static org.assertj.core.api.Assertions.assertThat;
-import static org.mockito.Mockito.times;
-import static org.mockito.Mockito.verify;
-import static org.mockito.Mockito.when;
+import static org.mockito.Mockito.*;
```

Comment:
I would escape wild card imports even in tests.

Reply:
will fix it